components/link/stories/link.stories.js

```diff
@@ -70,5 +73,88 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
-Default.args = {};
+export const Default = (args, context) => html`
+	${window.isChromatic() ? ChromaticLinkGroup(args, context) : Template(args, context)}
+
+`;
+Default.storyName = "Primary";
+
+/*
+ * Stories for the "Docs" only.
+*/
+export const Secondary = Template.bind({});
+Secondary.args = {
+	variant: "secondary",
+	text: "Link using spectrum-Link--secondary"
+};
+Secondary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+Secondary.tags = ["autodocs", "!dev"];
+
+export const StaticWhite = Template.bind({});
+StaticWhite.args = {
+	staticColor: "white",
+	text: "Link using spectrum-Link--staticWhite"
```

Comment:
On the docs site, all the variants (aside from Primary and Secondary) have some "filler" text that isn't part of the <a> tag, this was a nice touch because you can easily see the difference between the styles for the regular text and the link. Is this something that's doable here? (We could also make a card for it if it seems like doing it outside of this PR is a better idea.)

![image](https://github.com/user-attachments/assets/8f1d0462-7537-473d-bef8-06b8a947630e)

Reply:
That's a really great idea, and I wasn't totally sure how to implement it. I created a `TemplateWithFillerText` for those other variants...

<img width="421" alt="Screenshot 2024-07-18 at 12 38 55 PM" src="https://github.com/user-attachments/assets/c9c10842-f3b8-4d07-8c1f-fbaa78f0973b">

The default story is still just bound to `Template`, so that shouldn't affect Chromatic runs. But what does this mean for people using these variants? If they try to use `QuietPrimary`, are they stuck with this filler text? If so, that might be a different PR. We have other stories/components that have multiple templates, so it should be ok right?

https://github.com/adobe/spectrum-css/pull/2884/commits/7cc52563c5249ed1336ba4bfcb6d95b98d72d760